Proposed changes to make reference to FAIR Principles

PeterParslow opened the pull request:
Contents of https://github.com/w3c/sdw/blob/gh-pages/proposals/SDWBP%20FAIR%20thoughts.asciidoc turned into handcrafted HTML* and inserted into three places in the Best Practices

*I'm becoming a fan of ASCIIdoc ;)

PeterParslow committed:
Proposed changes to make reference to FAIR Principles

Contents of https://github.com/w3c/sdw/blob/gh-pages/proposals/SDWBP%20FAIR%20thoughts.asciidoc turned into handcrafted HTML* and inserted into three places in the Best Practices

*I'm becoming a fan of ASCIIdoc ;)

PeterParslow commented:
Intended to implement https://github.com/w3c/sdw/issues/1290

PeterParslow committed:
Cite sources for the FAIR+ and Q-FAIR ideas

PeterParslow committed:
Mention FAIR as outcome of BPs that contribute

The SDW ones, the list in the FAIR section covers DWBP as well as SDWBP

Malvoz commented on bp/index.html:
```diff
@@ -4700,6 +4716,95 @@ <h2>Cross reference of use case requirements against best practices</h2>
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/">R1.2. (Meta)data are associated with detailed provenance</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataProvenance>DWBP Best Practice 5: Provide data provenance information</a> "Provide complete information about the origins of the data and any changes you have made."
```
The DWBP Best Practice 5 does not get rendered when I view this locally. It looks like the `<a>` tag was closed improperly.

PeterParslow replied:
It was the href string which wasn't closed

Malvoz commented on bp/index.html:
```diff
@@ -587,6 +587,7 @@ <h2>Introduction</h2>
         are discoverability, accessibility and interoperability. Our overarching goal is to enable
         <a>spatial data</a> to be integrated within the wider Web of data; providing standard patterns and
         solutions that help solve these problems.</p>
+	<p>Following these guidelines should result in your data fitting more with the <a xlink:href="https://www.go-fair.org/fair-principles/">FAIR Principles</a>.</p>
```
`xlink:href` should be `href`.

Malvoz commented on bp/index.html:
```diff
@@ -661,7 +662,16 @@ <h3>Data publication</h3>
-
+      <section id="scope-FAIR">
+	<h3>FAIR Principles</h3>
+	  <p>The FAIR Principles are described at <a xlink:href="https://www.go-fair.org/fair-principles/">FAIR Principles - GO FAIR</a>; they are widely adopted (or at least aimed for)
```
`xlink:href` should be `href`.

PeterParslow replied:
Whoops - the other commits I tested offline (Notepad & browser), but I forgot this one!

Malvoz commented on bp/index.html:
```diff
@@ -4700,6 +4716,95 @@ <h2>Cross reference of use case requirements against best practices</h2>
+<p>
+The crawlers that search engine providers use to index the web are trained to look for human readable information. So the more “human readable” your metadata is, the more likely you are to be found in web searches – whether by humans or machines.
+</p><p>
+When publishing data on the web, take account of web accessibility guidelines. It is currently challenging to make web visualisations of geospatial data accessible to assistive technology. “Maps on the Web” is looking at this?
```
> “Maps on the Web” is looking at this?

It's not clear to me what "Maps on the Web" means in this context, the [Maps for the web](https://www.w3.org/2020/maps/) workshop was hosted in cooperation with the W3C [Maps for HTML](https://www.w3.org/community/maps4html/) Community Group. Should it reference the CG?

PeterParslow replied:
yes, thanks.

(I originally prepared this as a discussion document for the SDWWG; they then encouraged me to convert into specific pull requests, rather than discussing it first - so some "discussion" bits have slipped through)

Malvoz commented on bp/index.html:
```diff
@@ -4700,6 +4716,95 @@ <h2>Cross reference of use case requirements against best practices</h2>
+	    <i>I’m not so sure of the match(es) here</i>
+
+<h3>FAIR Challenges/extensions</h3>
+<h4>A – human accessibility</h4>
```
I'd like to think the [Towards Accessible Annotations for a Native Map Viewer for the Web Platform](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Nicolo_Carpignoli-Maps-for-the-Web.pdf) User needs and Requirements by Nicolò Carpignoli (W3C APA Invited Expert) and Joshue O Connor (W3C WAI) is an appropriate reference for the human accessibility section (as stated in https://github.com/w3c/sdw/issues/1311).

PeterParslow replied:
It does, for the specific case of presenting maps on the web. Happy to cite it.

Malvoz commented on bp/index.html:
```diff
@@ -4700,6 +4716,95 @@ <h2>Cross reference of use case requirements against best practices</h2>
+<p>
+The crawlers that search engine providers use to index the web are trained to look for human readable information. So the more “human readable” your metadata is, the more likely you are to be found in web searches – whether by humans or machines.
+</p><p>
+When publishing data on the web, take account of web accessibility guidelines. It is currently challenging to make web visualisations of geospatial data accessible to assistive technology. “Maps on the Web” is looking at this?
```
> web accessibility guidelines

Is this specifically referring to the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG/)? If so it could link to it (or https://www.w3.org/WAI/standards-guidelines/wcag/).

PeterParslow committed:
Remove a few xlink: that I'd left in in the FAIR bits

PeterParslow committed:
Close href string

PeterParslow committed:
Web-accessible source for part of FAIR+

Cite Towards Accessible Annotations for a Native Map Viewer for the Web Platform: User Needs and Requirements

PeterParslow committed:
Crosslink Maps for HTML Community Group

PeterParslow committed:
link to Web Content Accessibility Guidelines

PeterParslow commented:
Thanks @Malvoz  / Robert

prushforth committed:
Suggested changes to section on FAIR+

prushforth reviewed (dismissed):
I offer a [suggested revision](https://github.com/w3c/sdw/pull/1335) to the appendix on FAIR+ suggestions

KoalaGeo committed:
Add reference to Fair into Reality report

https://ec.europa.eu/info/sites/default/files/turning_fair_into_reality_0.pdf

KoalaGeo committed:
Use DOI URL

KoalaGeo committed:
Update bpconfig.js

Add reference

KoalaGeo committed:
Update bpconfig.js

KoalaGeo committed:
Add https://doi.org/10.15497/rda00050

prushforth committed:
Merge pull request #1335 from prushforth/PeterParslow-patch-6

Suggested changes to section on FAIR+

PeterParslow committed:
Merge branch 'PeterParslow-patch-6' into patch-1

PeterParslow committed:
Merge pull request #1336 from KoalaGeo/patch-1

Add reference to Fair into Reality report

jvanulde commented on bp/index.html:
```diff
@@ -4700,6 +4716,103 @@ <h2>Cross reference of use case requirements against best practices</h2>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#ReuseVocabularies">DWBP Best Practice 15: Reuse vocabularies, preferably standardized ones</a> "Use terms from shared vocabularies, preferably standardized ones, to encode data and metadata." This addresses more of the semantic standardization that exists in some community standards: "Use of vocabularies already in use by others captures and facilitates consen ...
+
+	    <i>I’m not so sure of the match(es) here</i>
```
Should this be removed?

Malvoz replied:
Otherwise make this an [`ednote`](https://github.com/w3c/respec/wiki/ednote)?

jvanulde commented on bp/index.html:
```diff
@@ -4700,6 +4716,103 @@ <h2>Cross reference of use case requirements against best practices</h2>
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/">R1.2. (Meta)data are associated with detailed provenance</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataProvenance">DWBP Best Practice 5: Provide data provenance information</a> "Provide complete information about the origins of the data and any changes you have made."
```
Missing a "See"?

jvanulde commented on bp/index.html:
```diff
@@ -4700,6 +4716,103 @@ <h2>Cross reference of use case requirements against best practices</h2>
+</li>
+<li><a href="https://www.go-fair.org/fair-principles/r1-1-metadata-released-clear-accessible-data-usage-license/">R1.1. (Meta)data are released with a clear and accessible data usage license</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataLicense">DWBP Best Practice 4: Provide data license information</a>; again, this doesn’t say that the license should be "clear", but it does say that it should be accessible - "attached to data" (via a link from the metadata or embedded in the metadata).
```
Missing a "See"?

jvanulde reviewed (approved):
Suggestions for consideration.

prushforth commented on bp/index.html:
```diff
@@ -661,7 +662,16 @@ <h3>Data publication</h3>
+	    there is a lot of overlap between the FAIR Principles and the best practices described in this paper.</p>
+	  <p>Similarly, although not currently expressed in terms of the FAIR Principles,
+	    the Data on the Web Best Practices are also designed to make it easier for "data consumers to find, use and link to the data".</p>
+	  <p>There have also been some suggestions for improvement on the FAIR principles, and these are also discussed <a href="#FAIRPrinciples" class="sectionRef"></a>.</p>
```
Would this refer to the section entitled "FAIR Challenges/extensions" (doesn't have an id)

prushforth commented on bp/index.html:
```diff
@@ -1295,6 +1305,7 @@ <h4 class="subhead">Why</h4>
           <section class="outcome">
             <h4 class="subhead">Intended Outcome</h4>
             <p>Spatial Things become part of the Web’s global information space enabling them be linked with other <a>Spatial Things</a> and other resources and for those <a>links</a> to be durable. In other words, <a>spatial data</a> becomes part of the Web of Data.</p>
+		  <p>Your data is more <a href="https://www.go-fair.org/fair-principles/">findable</a></p>
```
I don't believe this link is necessary.  There are other labelled links that lead to this web site, so this is redundant.

prushforth commented on bp/index.html:
```diff
@@ -1401,6 +1412,7 @@ <h4 class="subhead">Intended Outcome</h4>
             <p>Information about spatial datasets and things is indexed by search engines.</p>
             
             <p>Users can find <a>Spatial Things</a> using common search engines.</p>
+	    <p>That is, your data is more <a href="https://www.go-fair.org/fair-principles/">findable</a></p>
```
Unnecessary link.

prushforth commented on bp/index.html:
```diff
@@ -1579,6 +1591,7 @@ <h4 class="subhead">Why</h4>
             <h4 class="subhead">Intended Outcome</h4>
             <p><a>Links</a> can be identified and traversed by humans and software agents.</p>
             <p>Sufficient information is provided to help humans and software agents determine whether traversal of a given link meets their goals.</p>
+	    <p>Your data is more <a href="https://www.go-fair.org/fair-principles/">interoperable</a></p>
```
Redundant link

prushforth commented on bp/index.html:
```diff
@@ -1697,6 +1710,7 @@ <h4 class="subhead">Why</h4>
             <section class="outcome">
               <h4 class="subhead">Intended Outcome</h4>
               <p><a>Spatial data</a> can be used easily and reliably by the target users.</p>
+	      <p>Your data is more <a href="https://www.go-fair.org/fair-principles/">interoperable</a></p>
```
Redundant link

prushforth commented on bp/index.html:
```diff
@@ -2849,7 +2863,8 @@ <h4 class="subhead">Why</h4>
               <h4 class="subhead">Intended Outcome</h4>
               <p>Spatial things are related to other resources in the Web of data using links with appropriate semantics.</p>
-            </section>
+	      <p>Your data is more <a href="https://www.go-fair.org/fair-principles/">interoperable</a></p>
```
Redundant link.

prushforth commented on bp/index.html:
```diff
@@ -3795,6 +3810,7 @@ <h4 class="subhead">Intended Outcome</h4>
                 within catalog services such as those provided by <a>SDIs</a>.</li>
               <li>Dataset metadata should also include the information required for a user to evaluate
                 whether a spatial dataset is suitable for their intended application.</li>
+	      <li>Your data is more <a href="https://www.go-fair.org/fair-principles/">findable</a></li>
```
Duplicate link

Commit:
Resolve comments I made in review. Fix some grammatical errors I created.

prushforth commented:
OK I pushed what I believe was necessary, reviewed and approved from me.

PeterParslow commented:
> OK I pushed what I believe was necessary, reviewed and approved from me.

"pushed" is new to me, and the "pull request" remains to be "merged"
